dags.yaml

```diff
@@ -378,3 +378,19 @@ bqetl_desktop_platform:
       ]
     retries: 2
     retry_delay: 30m
+
+bqetl_mozregression:
```

Comment:
I'd recommend adding a `description` field (brand new feature!) with additional context about the intent of the DAG and perhaps links to the mozregression docs.

Reply:
There are some examples in the `dags.yaml` file for reference: https://github.com/mozilla/bigquery-etl/blob/350291bd1a9608d6659f6cbd4c9123f4a366434f/dags.yaml#L91